iceberg/iceberg-handler/pom.xml

```diff
@@ -106,6 +106,12 @@
             <artifactId>assertj-core</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.roaringbitmap</groupId>
```

Comment:
How come we need this here? I thought this would come transitively via iceberg-core

Reply:
I think this is due to roaringbitmap having a 'runtime' scope in the iceberg-core pom:
https://repo1.maven.org/maven2/org/apache/iceberg/iceberg-core/0.13.0/iceberg-core-0.13.0.pom

And we do need it on classpath for tests too.